entries/a/aero.zip.json

```
@@ -0,0 +1,12 @@
+{
+  "aero.zip": {
+    "mfa": "allowed",
+    "passwordless": "allowed",
+    "documentation": "https://aero.zip/faq/can-i-log-in-with-passkey",
+    "notes": "Passwordless login only works with passkeys that support PRF.",
+    "categories": [
+      "backup",
+      "creativity"
```

Thanks for the pull request @pypt!
My only change request is the removal of the creativity category. That category is meant for audio, video and image editing tools. As aero.zip only transfers files, it's not a proper fit for said category. Backup & Sync is good though :+1:
```suggestion
      "backup"
```